xml/art_minvm.xml

```diff
@@ -97,24 +97,24 @@
   <sect2 xml:id="sec-system-requirements">
    <title>System requirements</title>
    <para>
-    The minimum &jeos; system requirements are similar to &sls;. For more
+    The minimum &minvm; system requirements are similar to &sls;. For more
     information, refer to <xref
    linkend="sec-x86-requirements"/>.
    </para>
   </sect2>
 
-  <sect2 xml:id="sec-jeos-sles-diff">
-   <title>Differences between &jeos; and &sls;</title>
+  <sect2 xml:id="sec-minvm-sles-diff">
+   <title>Differences between &minvm; and &sls;</title>
    <para>
-    There are several significant differences between &jeos; and &sls;.
+    There are several significant differences between &minvm; and &sls;.
    </para>
-   <variablelist xml:id="vl-jeos-sles-diff">
-    <title>Differences between &jeos; and &sls;</title>
+   <variablelist xml:id="vl-minvm-sles-diff">
+    <title>Differences between &minvm; and &sls;</title>
     <varlistentry>
      <term>jeos-firstboot</term>
      <listitem>
       <para>
-       &jeos; comes with the <package>jeos-firstboot</package> tool, which can
+       &minvm; comes with the <package>jeos-firstboot</package> tool, which can
```

Comment:
It the name `jeos-firstboot` still the same? Or has the name changed as well?

Reply:
[This page](https://confluence.suse.com/display/jeos/jeos+packages+renamed) says it's still `jeos-firstboot`. Not sure if it's up-to-date.

Reply:
Thanks for the link. The last change was from 2022-02-02, so it's up-to-date.

Dima, the page was changed by Vincent, maybe ask him for clarification?

Reply:
it's still jeos-firstboot for 15SP4, we recently got feedback that MicroOS might use this in the future... so we might renamed it in the future (sic) but not for 15SP4